src/app/dashboard/AvailableRooms.jsx

```diff
@@ -46,12 +45,14 @@ function RoomModal({ room, onJoinRoomClick, onDeleteClick, onClose }) {
                 p={2}
                 sx={{
                     border: '2px solid grey',
-                    left: '5vw',
                     position: 'relative',
                     textAlign: 'center',
+                    backgroundColor: '#f0f0f0',
+                    borderRadius: 2,
+                    color: grey[900],
                 }}
                 >
-                {room.passcode && <p>Passcode: {room.passcode}</p>}
+                {room._id && <p>Link: localhost:3000/challenge/{room._id}</p>}
```

Comment:
Please adjust this to reference a generic url - in production, it won't be `localhost:3000`.